umbral/_pre.py

```diff
@@ -155,8 +161,17 @@ def verify_kfrag(kfrag: 'KFrag',
     #  We check that the commitment u1 is well-formed
     correct_commitment = u1 == key * u
 
-    kfrag_validity_message = bytes().join(
-        bytes(material) for material in (id, delegating_point, receiving_point, u1, ni, xcoord))
+    pubkey_size = UmbralPublicKey.expected_bytes_length(curve=params.curve)
+
+    if delegating_pubkey is None:
```

Comment:
I think I need a quick explanation of this - we use zeroed-out keys if none are provided?  I'll just ask on Discord.